katrain/popups.kv

```diff
@@ -753,14 +753,17 @@
             LabelledCheckBox:
                 id: fast
                 size_hint: 0.05,1
+                input_property: "general/load_fast_analysis"
+                on_press: root.update_config(True)
```

Comment:
could we do this only on submit?

Reply:
you mean on load file?

Reply:
yep, just when the dialog is submitted. a little more consistent.

Reply:
i am not familiar with the dialog event ,
when debugging ,the on_submit method was not invoked. any idea?

Reply:
done